### PR TITLE
added a new cmdlet Get-LicenseReportForChangeNotificationSubscription

### PR DESCRIPTION
**Pull request by anandab-msft**

### CODE REVIEW

**Comment by garycentric** [comment]
@anandab-msft, @Kateyanne 

For this content to build correctly, the file name, _Get-LicenseReportForChangeNotificationSubscription_, must end with the Markdown extension, _.md_. So, you need to rename the file.

Also, it's my understanding that PowerShell _reference_ content should not use our Markdown extensions, like `> [!NOTE]`. Instead, you'll need to use `NOTE:` and basic formatting.

Finally, the URL to docs.microsoft.com in "Synopsis" should have **/en-us** removed. This is standard practice for links to docs.microsoft.com so that locale settings of a user's browser are not ignored. Also, that sentence in "Synopsis" should end with a period.

**Comment by garycentric** [comment]
@anandab-msft 

When you've addressed the blocking issues, please remove _Back to submitter_ and re-apply _Sign off_ to put this PR back in the queue for a follow-up review.

**Comment by garycentric** [comment]
@anandab-msft 

Sorry, but I need to amend one of my comments from earlier: rather than remove **/en-us** from the URL in "Synopsis", that link needs to be changed to a site-relative link. So, the link should be:

`This cmdlet tells whether a user has the required license to export their messages via [change notification subscription](/graph/teams-licenses).`

Also, the same link is repeated in "Description":

`For more details please review [Licenses for subscribing to chat messages](/graph/teams-licenses).`

These links might not work in the preview version for this PR (a link for which should be generated after you add the file extension to the file name), but they'll work when the article is merged to master. You can test this by changing **branch=pr-en-us-7707** in the preview URL to **branch=master**.

File: teams/teams-ps/teams/Get-LicenseReportForChangeNotificationSubscription [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+manager: alagra
+online version: https://docs.microsoft.com/powershell/module/teams/get-licensereportforchangenotificationsubscription 
+schema: 2.2.0-preview
+---
+
+# Get-LicenseReportForChangeNotificationSubscription
+
+## SYNOPSIS
+
+This cmdlet tells whether a user has the required license to export their messages via [change notification subscription](https://docs.microsoft.com/en-us/graph/teams-licenses)
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-LicenseReportForChangeNotificationSubscription:16]:
```suggestion
This cmdlet tells whether a user has the required license to export their messages via [change notification subscription](/graph/teams-licenses).
```

File: teams/teams-ps/teams/Get-LicenseReportForChangeNotificationSubscription [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## SYNTAX
+
+```
+Get-LicenseReportForChangeNotificationSubscription [-Period] <Number> 
+```
+
+## DESCRIPTION
+
+This cmdlet supports retrieving the total number of messages sent by a user in chat/channel and whether a user has the required license(s) to sent change notification events when subscribed for chat or channel messages. For more details please review https://docs.microsoft.com/en-us/graph/teams-licenses
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-LicenseReportForChangeNotificationSubscription:26]:
```suggestion
This cmdlet supports retrieving the total number of messages sent by a user in chat/channel and whether a user has the required license(s) to send change notification events when subscribed for chat or channel messages. For more details, please review [Licenses for subscribing to chat messages](/graph/teams-licenses).
```

File: teams/teams-ps/teams/Get-LicenseReportForChangeNotificationSubscription [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+> [!NOTE]
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-LicenseReportForChangeNotificationSubscription:28]:
```suggestion
```

File: teams/teams-ps/teams/Get-LicenseReportForChangeNotificationSubscription [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+> This cmdlet is currently supported in preview version only.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-LicenseReportForChangeNotificationSubscription:29]:
```suggestion
This cmdlet is currently supported in preview version only.
```

File: teams/teams-ps/teams/Get-LicenseReportForChangeNotificationSubscription [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+## EXAMPLES
+
+### Example
+```powershell
+PS C:\> Get-DlpUserLicenseReport -Period 7
+```
+
+Returns license info and total messages sent by users in last 7 days.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-LicenseReportForChangeNotificationSubscription:38]:
```suggestion
Returns license info and total messages sent by users in the last 7 days.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please also add the Markdown extension, .md to the filename as per @garycentric's suggestion.

File: teams/teams-ps/teams/Get-LicenseReportForChangeNotificationSubscription (hunk 2 above)
Comment (anandab-msft) [reply teams/teams-ps/teams/Get-LicenseReportForChangeNotificationSubscription:26]: @JohanFreelancer9 - the license information page in different repo , hope relative link will work in that case as well.

**Comment by JohanFreelancer9** [comment]
@anandab-msft - thanks for committing the suggestions. I have actually just checked and it seems the relative link does work. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by anandab-msft** [comment]
> @anandab-msft
> 
> When you've addressed the blocking issues, please remove _Back to submitter_ and re-apply _Sign off_ to put this PR back in the queue for a follow-up review.

can you pls change the label, i don't see an option to do so

**Comment by traya1** [comment]
closing/re-opening PR. Staging is not reflecting proposed edits.

**Comment by v-dihans** [comment]
@anandab-msft  

Thanks for the fixes. Merging now.

**Pull request merged**